include/pt-mbr.h

```diff
@@ -2,34 +2,35 @@
 #define UTIL_LINUX_PT_MBR_H
 
 #include <assert.h>
+#include <inttypes.h>
 
 struct dos_partition {
-	unsigned char boot_ind;		/* 0x80 - active */
-	unsigned char bh, bs, bc;	/* begin CHS */
-	unsigned char sys_ind;
-	unsigned char eh, es, ec;	/* end CHS */
-	unsigned char start_sect[4];
-	unsigned char nr_sects[4];
+	uint8_t boot_ind;	/* 0x80 - active */
+	uint8_t bh, bs, bc;	/* begin CHS */
+	uint8_t sys_ind;
+	uint8_t eh, es, ec;	/* end CHS */
+	uint8_t start_sect[4];
+	uint8_t nr_sects[4];
 } __attribute__((packed));
 
 #define MBR_PT_OFFSET		0x1be
 #define MBR_PT_BOOTBITS_SIZE	440
 
-static inline struct dos_partition *mbr_get_partition(unsigned char *mbr, int i)
+static inline struct dos_partition *mbr_get_partition(uint8_t *mbr, int i)
 {
```

Comment:
I don't like this change. It's better to keep MBR as unsigned char * as it's expected everywhere and for things like mbr + MBR_PT_OFFSET it makes more sense for readers than use it as ints array.

IMHO it would be better to reject all this patch :-)

Reply:
Dropped as well. And the branch is rebased on top of more recent origin/master.